### PR TITLE
Add photo support

### LINKED ISSUE #10: Image support for scrapper
Update live_migration.py to include images for each listing by scraping the HTML.

### PR DESCRIPTION
**Pull request by fatlotus**
This pull request adds migration, serving, and expiration, and fixes #10.

### CODE REVIEW

File: vendor/cloudstorage/api_utils.py [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+import httplib
+import logging
+import math
+import os
+import threading
+import time
+import urllib
+
+
+try:
```
Comment (georgeteo) [inline vendor/cloudstorage/api_utils.py]: Why are these imports in a try-except control flow.

**Comment by georgeteo** [comment]
Should I read through the GAE Cloud Storage package import too?

File: vendor/cloudstorage/api_utils.py (hunk 1 above)
Comment (fatlotus) [inline vendor/cloudstorage/api_utils.py]: Dunno... this isn't code I wrote, it's just imported.

**Comment by fatlotus** [comment]
If you want :/ It's not code I wrote, though.

**Comment by georgeteo** [comment]
Okay. Yay frameworks!

**Pull request merged**